pkg/auth/authenticator/password/htpasswd/htpasswd.go

```diff
@@ -194,10 +197,11 @@ func testMD5Password(password, hash string) (bool, error) {
 }
 
 func testCryptPassword(password, hash string) (bool, error) {
-	// if len(password) > 8 {
```

Comment:
I think you'd need to distinguish between the DES and MD5 cases here, in order to pass in the correct salt